sessions/class-01/exercise/start.js

```diff
@@ -81,4 +108,11 @@ export function getFirstAndLastNames(name) {}
  * @param {any} def Default value.
  * @returns {any} Property value.
  */
-export function getValue(obj, key, def) {}
+export function getValue(obj, key, def) {
+    if(!obj){
+        return def;
+    }else{
+        return obj[key] || def
+    }
```

Comment:
This was tricky and I intentionally left wrong code in the master. A proper implementation would be

```js
export function getValue(obj, key, def) {
	if (!hasValue(obj)) {
		return def;
	}
	// If the obj[key] is undefined, then return default,
	// else return obj[def]
	return obj[key] === undefined ? def : obj[key];
}
```

And to make the test account for that

```js
expect(getValue({ foo: false }, 'foo', 'def')).toBe(false);
```